src/components/common/CustomPicker/styles.js

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+
+import { regularFontSize, black } from '../../../constants/styleConstants';
+
+const styles = StyleSheet.create({
+  label: {
+    fontSize: regularFontSize,
+    alignSelf: 'center'
+  },
+
+  inputBox: {
+    borderColor: black,
+    borderWidth: 1,
+    borderStyle: 'solid',
+    height: 37,
+    margin: 10,
+    width: 188,
+    alignSelf: 'center'
+  },
+
+  pickerStyle: {
+    fontSize: 16,
+    paddingTop: 13,
+    paddingHorizontal: 10,
+    paddingBottom: 12,
+    borderWidth: 1,
+    borderColor: 'gray',
```

Comment:
Put `'gray'`, `'white'` and  `'black'` as constants in your stylesConstants file with their hex values if you haven't already and import them here.

Reply:
Yeah, totally missed that one.